content/docs/command-reference/add.md

```diff
@@ -81,10 +81,8 @@ copy of the target data directly to a remote of your choice (or the default
 one). A `.dvc` file will be created normally, but the data won't be found in
 your local project until you `dvc pull` it.
 
-This option is useful when the local system can't handle the target data, but
-you still want to track and store it in remote storage, so that whenever you
-switch to a different system that can handle it, you can simply pull the data
-and start working on it.
+(ℹ️) See the [Transfer to remote storage](#example-transfer-to-remote-storage)
+below.
```

Comment:
Should we remove the **Transferring data directly to remote storage** section since just one paragraph is left in it? And maybe make the sentence with the link to the example part of that paragraph.

Reply:
I'd even try to summarize that previous paragraph a little bit... Suggestion:

```md
Note that `dvc add` supports `targets` external to the <abbr>workspace</abbr>
(see `--external`), but often external data is too large to be
<abbr>cached</abbr> locally. To track the data without downloading it, you can
use the `--to-remote` option, which will transfer a copy directly to
[remote storage](/doc/command-reference/remote). The data won't be found in your
local project until you `dvc pull` it. See the
[Transfer to remote storage](#example-transfer-to-remote-storage) below.
```

Reply:
But TBH I'm not sure we want to have this paragraph at all since the `add` Description is already so long. It's already mentioned a bit earlier (shown below) and the `--to-remote` will link to the Example, where it's all explained. Idk. WDYT @shcheklein ?

![image](https://user-images.githubusercontent.com/1477535/107550515-c045d300-6b96-11eb-96ba-ec601e8db942.png)

Reply:
> p.s. that earlier "remote storage" text probably should link to /doc/command-reference/remote though 🙂

Reply:
On this I guess the shorter the better so I vote to scratch the entire new paragraph and instead just mention in step 4. that the .dvc file created with --to-remote changes the external path to a local path.